examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -135,6 +135,7 @@ class CupertinoDemoTab1 extends StatelessWidget {
   Widget build(BuildContext context) {
     return CupertinoPageScaffold(
       child: CustomScrollView(
+        semanticChildren: 50,
```

Comment:
semanticsChildrenCount? children sounds like I can pass in the actual children?

Reply:
Not sure about semanticChildrenCount vs semanticsChildrenCount

Reply:
uber nitty nit: Almost everywhere else we say "childCount" rather than "childrenCount". If we insist on "children", then perhaps we should say "numberOfChildren".